packages/terra-docs/tests/wdio/button-spec.js

```diff
@@ -0,0 +1,27 @@
+Terra.describeViewports('Button', ['medium'], () => {
+  describe('Default Button', () => {
+    it('checks accessibility', () => {
+      browser.url('/raw/tests/terra-docs/docs/button');
+      browser.moveToObject('#root', 700, 700);
+      Terra.validates.element('checks accessibility-default', { selector: '#site' });
+    });
+
+    it('checks keyboard focus', () => {
+      browser.url('/raw/tests/terra-docs/docs/button');
+      browser.keys('Tab');
+      Terra.validates.element('Checks keyboard focus', { selector: '#site' });
+      browser.keys('Space');
+      Terra.validates.element('Checks keyboard focus - Space', { selector: '#site' });
+      browser.keys('Enter');
+      Terra.validates.element('Checks keyboard focus - Enter', { selector: '#site' });
+    });
+
+    it('checks mouse interactions', () => {
+      browser.url('/raw/tests/terra-docs/docs/button');
+      browser.moveToObject('#button');
+      Terra.validates.element('checks mouse interactions - hover', { selector: '#site' });
+      browser.buttonDown();
+      Terra.validates.element('checks mouse interactions - active', { selector: '#site' });
```

Comment:
I'd break this into a new test. if `Terra.validates.element('checks mouse interactions - hover', { selector: '#site' });` fails for mis-match, this test isn't ran & you have to run 2x to get all diffs.

Reply:
This seems fine to me. The test runs through the workflow of hovering and then clicking and we validate along the way. I don't think we should have the expectation of one screenshot per test.